chore: docs rework (#927)
open-feature/flagd: docs/reference/specifications/custom-operations/fractional-operation-spec.md

# Fractional Operation Specification

This evaluator allows to split the returned variants of a feature flag into different buckets,
where each bucket can be assigned a percentage, representing how many requests will resolve to the corresponding
variant.

> The sum of all weights must be 100, and the distribution must be performed by using the value of a referenced

Reviewer: ```suggestion
The sum of all weights must be `100`, and the distribution must be performed by using the value of a referenced
```

This makes it a bit easier to parse out of the sentence.

The remaining items are `arrays`, each with two values, with the first being `string` item representing the name of the variant, and the
second being a `float` item representing the percentage for that variant. The percentages of all items must add up to
100.0, otherwise unexpected behavior can occur during the evaluation. The `data` object can be an arbitrary

> JSON object. Below is an example for a targetingRule containing a `fractional`:

Reviewer: ```suggestion
JSON object. Below is an example of a targeting rule containing a `fractional`:
```
